lib/tasks/geocode.rake

```diff
@@ -0,0 +1,25 @@
+namespace :geocode do
+  desc 'Geocode candidate addresses'
+  task candidate_addresses: :environment do
+    if Geocoder.config.api_key.blank?
+      raise 'You need to set a GOOGLE_MAPS_API_KEY'
+    end
+
+    Benchmark.bm do |x|
```

Comment:
LGTM ... Are there any implications for when it's run in prod in terms of performance or affecting users?

Reply:
Shouldn't be anything too heavy, there's only ~4300 records in scope for his. Will result in about 4 minutes of increased database IO, if early benchmarks are any indication.

Reply:
I guess so long as we have 11 spare Postgres connections and it's run in a container other than the Web application  it would be ok.